### DISCUSSION TITLE
Setting both PathParam and HeaderParam with the same value in quarkus rest client

### DISCUSSION THREAD

**Discussion by suchendra** [quarkusio/quarkus#27927] [Q&A] [answered]
Is it possible to set the same value for both header as well as for path parameter in quarkus rest client ? 
Something like below;

`Response getBookDetails(@PathParam("orgId")  @HeaderParam("orgId") String org);` 

I was thinking about this,

`Response getBookDetails(@PathParam("orgId") String orgId, @HeaderParam(orgId) String org);`

But the caller has to repeat the arguments.

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
That's not really possible. 
Although I do see it could make sense (for the client), it's not something that is supported. 

Also I think supporting it would make things slight more complicated for people to understand - now it's very simple: one annotation per parameter method.
